lib/diagnostic.js

```diff
@@ -30,16 +33,24 @@ const sum = function (previous, current) {
 // for every element in the array returns `false`
 // you may use any feature of JavaScript
 // to solve this problem
-const hasNone = function (array, callback) {
 
+const hasNone = function (array, callback) {
+    const isFalse = array.filter(hasNone = function(array) {
+    if ((callback(element, i, originalArray) === false)) {
+  }
```

Comment:
You want to make sure that a function you are passing to `filter` either returns `true` or `false`